redis/howto/connectclient.mdx

```diff
@@ -69,7 +65,7 @@ const redis = new Redis({
 ```javascript
 const Redis = require("ioredis");
 
-let client = new Redis("redis://:YOUR_PASSWORD@YOUR_ENDPOINT:YOUR_PORT");
+let client = new Redis("rediss://:YOUR_PASSWORD@YOUR_ENDPOINT:YOUR_PORT");
```

Comment:
why double?

Reply:
This means the TLS is enabled in the redis connection